### PR TITLE
Add section about vote privacy

### PR DESCRIPTION
**Pull request by Nutomic**
Does the text sound good? And should we move this section higher up in the file?

### CODE REVIEW

File: src/users/03-votes-and-ranking.md [hunk 1]
```diff
@@ -35,3 +35,7 @@ Comments can be sorted in the following ways. These all keep the indentation int
 Additionally there is a sort option **Chat**. This eliminates the hierarchy, and puts all comments on the top level, with newest comments shown at the top. It is useful to see new replies at any point in the conversation, but makes it difficult to see the context.
 
 The ranking algorithm is described in detail [here](../contributors/07-ranking-algo.md).
+
+# Vote Privacy
+
+Lemmy attempts to limit the visibility of votes to protect user privacy. But due to the way Lemmy works, votes cannot be completely private. Both instance admins and community moderators can see the names of users who voted on a given post or comment. This helps to fight against vote manipulation. Additionally, individual votes are federated over ActivityPub together with the corresponding username. This means that other federated platforms can freely choose how to display vote information, even going as far as publicly displaying individual votes.
```
Comment (dessalines) [inline src/users/03-votes-and-ranking.md:41]: > Both instance admins and community moderators can see the names of users who voted on a given post or comment.

Change to:

`Instance admins can see the names of everyone who voted on a given post or comment, and and community moderators can see the names for the communities they moderate.`

Comment (Nutomic) [reply src/users/03-votes-and-ranking.md]: Done

File: src/users/03-votes-and-ranking.md [hunk 2]
```diff
@@ -38,4 +38,4 @@ The ranking algorithm is described in detail [here](../contributors/07-ranking-a
 
 # Vote Privacy
 
-Lemmy attempts to limit the visibility of votes to protect user privacy. But due to the way Lemmy works, votes cannot be completely private. Both instance admins and community moderators can see the names of users who voted on a given post or comment. This helps to fight against vote manipulation. Additionally, individual votes are federated over ActivityPub together with the corresponding username. This means that other federated platforms can freely choose how to display vote information, even going as far as publicly displaying individual votes.
+Lemmy attempts to limit the visibility of votes to protect user privacy. But due to the way Lemmy works, votes cannot be completely private. Instance admins can see the names of everyone who voted on a given post or comment, and and community moderators can see the names for the communities they moderate. This helps to fight against vote manipulation. Additionally, individual votes are federated over ActivityPub together with the corresponding username. This means that other federated platforms can freely choose how to display vote information, even going as far as publicly displaying individual votes.
```
Comment (dessalines) [inline src/users/03-votes-and-ranking.md:41]: `and and`

Comment (Nutomic) [reply src/users/03-votes-and-ranking.md]: copy-pasted from you :p

fixed now

**Review by dessalines — APPROVED** [review]

**Pull request merged**